packages/node_modules/@node-red/editor-client/src/js/ui/tab-info-outliner.js

```diff
@@ -401,6 +401,52 @@ RED.sidebar.info.outliner = (function() {
         updateSearch();
     }
 
+    function updateConfigUserCount(candidates) {
+        var types = globalConfigNodes.children;
+        if (types) {
+            var configMap = {};
+            types.forEach(function (type) {
+                var nodes = type.children;
+                if (candidates[type.label] && nodes) {
+                    nodes.forEach(function (node) {
+                        configMap[node.id] = node;
+                    });
+                }
+            });
+            RED.nodes.eachConfig(function (conf) {
```

Comment:
Iterating over the list of all config nodes every time a node is modified is going to be very inefficient. It should be possible to make this more targeted and only touch nodes that need updating.